Project.toml

```diff
@@ -48,7 +48,7 @@ SciMLBase = "1.8"
 Setfield = "0.7"
 SpecialFunctions = "0.7, 0.8, 0.9, 0.10, 1.0"
 StaticArrays = "1.1"
-SymbolicUtils = "0.13"
```

Comment:
Why did you remove SU?

Reply:
Whoops. I've added my SU branch locally to develop. Before merging this PR, my SymbolicUtils.jl PR should be merged (or used in this PR branch), because otherwise the TermInterface.jl integration won't work. Re-adding it now.

Is there any way to specify in Project.toml that i want SymbolicUtils.jl from latest commit in `ale/terminterface` branch? I can manually do `]add https://github.com/JuliaSymbolics/SymbolicUtils.jl#ale/terminterface`, but when the dependencies are re-instantiated it tries to pull from registry. My bad.

Reply:
https://discourse.julialang.org/t/does-manifest-toml-belong-in-the-repository/12029/13

What should I do then?